app/models/rbac.rb

```diff
@@ -124,6 +124,16 @@ def self.get_self_service_object_ids(user_or_group, klass)
     targets
   end
 
+  def self.calc_filtered_ids(klass, scope, user_filters, user_or_group = nil)
```

Comment:
The phrase `user_filters` is used throughout rbac. It is swapped to `rbac_filters` in a few instances and that confuses me.
I'd prefer to leave as `user_filters`.

Near future, I'd like this code to bubble up and be removed from rbac